src/core/server/logging/logging_system.ts

```diff
@@ -67,9 +71,12 @@ export class LoggingSystem implements LoggerFactory {
    * Updates all current active loggers with the new config values.
    * @param rawConfig New config instance.
    */
-  public upgrade(rawConfig: LoggingConfigType) {
-    const config = new LoggingConfig(rawConfig)!;
-    this.applyBaseConfig(config);
+  public async upgrade(rawConfig: LoggingConfigType) {
+    // We only want the console appender for the CLI process,
+    // so we use the 'default' configuration as defined by the schema.
+    const usedConfig = this.env.isDevCliParent ? loggingConfig.schema.validate({}) : rawConfig;
```

Comment:
can we keep this logic outside of logging system? we ca make `rawConfig?: LoggingConfigType` to make sure it falls back to the default config